### PR TITLE
Add message fixture for DynamicArrayStaticArrayPrimitivesNested

### PR DESCRIPTION
**Pull request by jacobperron**
Connects to 
This resolves CI failures introduced by #43.

CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/5361/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/2088/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/4451/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/5321/)

### CODE REVIEW

**Comment by jacobperron** [comment]
Warnings are from flake8, fixed by ros2/rmw_connext#305.

File: test_msgs/include/test_msgs/message_fixtures.hpp [hunk 1]
```diff
@@ -292,6 +293,22 @@ get_messages_dynamic_array_primitives_nested()
...
+std::vector<test_msgs::msg::DynamicArrayStaticArrayPrimitivesNested::SharedPtr>
+get_messages_dynamic_array_static_array_primitives_nested()
+{
+  std::vector<test_msgs::msg::DynamicArrayStaticArrayPrimitivesNested::SharedPtr> messages;
+
+  auto msg = std::make_shared<test_msgs::msg::DynamicArrayStaticArrayPrimitivesNested>();
+  std::vector<test_msgs::msg::StaticArrayPrimitives::SharedPtr> primitive_msgs =
+    get_messages_static_array_primitives();
+  for (size_t i = 0; i < primitive_msgs.size(); ++i)
+  {
```
Comment (clalancette) [inline test_msgs/include/test_msgs/message_fixtures.hpp]: I'm somewhat surprised the linters didn't pick this up, but our usual style is curly braces on the same line as the `for` statement.

**Review by clalancette — APPROVED** [review]
One nit to fix, but I'll approve anyway.

File: test_msgs/include/test_msgs/message_fixtures.hpp (hunk 1 above)
Comment (jacobperron) [reply test_msgs/include/test_msgs/message_fixtures.hpp]: There were no lint tests enabled for this package. I've enabled the linter and fixed issues.

**Comment by clalancette** [comment]
:+1:

**Comment by jacobperron** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/5367/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/2094/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/4457/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/5327/)

**Pull request merged**